test.js

```diff
@@ -1,7 +1,10 @@
 'use strict';
+const fs = require('fs');
+const path = require('path');
 const test = require('ava');
 const tempfile = require('tempfile');
 const readPkg = require('read-pkg');
+const writeJsonFile = require('write-json-file');
 const m = require('.');
```

Comment:
Is it a blocker? I can fix it tonight but I'd really need this published by tomorrow. I want to publish pnpm v1, with indentation detection